cli/find_in_files.php

```diff
@@ -51,7 +51,7 @@
 --filenames=filename            Cooma-separated list of file names to be searched.
 --skip-filenames=filename       Comma-separated list of file names to be omitted from the search.
 --skip-areas=areaname           Areas to skip. Separate multiple areas with a comma.
---openzips                      Open zip files, and search the files inside.
+--open-zips                      Open zip files, and search the files inside.
```

Comment:
Small indenting inconsistency

Reply:
Fixed, along with a couple of other typos while I was there